api/app.js

```diff
@@ -0,0 +1,9 @@
+const express = require('express');
+const app = express();
+const port = 3000;
+
+require('./db');
```

Comment:
You should create a db class and use it everywhere.
Please read how to use pool correctly.